### PR TITLE
Add popover and styling to prompt people to help us develop unsupported features

### LINKED ISSUE #80: make a <HelpUsImplementThisFeature/> component
As a developer
> I would like a React component that shows a tooltip of some sort over its child elements that requests help on implementing a feature so that I can easily specify that certain content in the app hasn't been made into a functional feature yet

Additionally this can link to a waffle card associated with the feature needed to be impletemed.

### PR DESCRIPTION
**Pull request by wdoug**
This PR closes #80

#### What does this PR do?

- Adds a `HelpUsImplementThisComponent` that styles can wrap arbitrary jsx and adds some styling and a popover on hover asking help on implementing that feature
- Uses that component on a few UI elements that aren't supported yet

screen shot 2018-01-26 at 11 48 57 am

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,18 +4,20 @@ import MemberResources from './MemberResources';
...
 import MemberProfileEdit from './MemberProfileEdit';
-import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import { MuiThemeProvider, createMuiTheme } from 'material-ui/styles';
 import EmailList from './EmailList';
 import './App.css';
 import { componentWithLoggedInUser } from './utils';
 import UsersList from './UsersList';
 import MemberProfilePage from './MemberProfilePage';
 
+const theme = createMuiTheme();
```
Comment (dviramontes) [inline src/App.js:14]: interesting, no need to call `new` here ?

**Comment by wdoug** [comment]
Note: You can preview this change [here](https://deploy-preview-116--cfd-members.netlify.com/volunteers/cjc244ll00sjl0186g1uzlre2)

File: src/presentational/HelpUsImplementThis.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const HelpUsImplementThis = ({
+  children,
+  featureDescriptionUrl = 'https://waffle.io/codefordenver/members'
+}) => {
+  return (
+    <Tooltip
+      className="HelpUsImplementThis"
+      title="Help us implement this feature"
+    >
+      <a href={featureDescriptionUrl} target="_blank" rel="noopener noreferrer">
```
Comment (dviramontes) [inline src/presentational/HelpUsImplementThis.js:14]: @wdoug what does `noreferrer ` attribute do here?

File: src/App.js (hunk 1 above)
Comment (wdoug) [reply src/App.js:14]: I don't think so. I pulled this from the new Material UI docs [here](https://material-ui-next.com/customization/themes/#examples). Also, per JS naming conventions, if it was expecting to be called with `new` I would have expected it to be `const theme = new MuiTheme();`, whereas `createMuiTheme` makes me think that it's just a function.

File: src/presentational/HelpUsImplementThis.js (hunk 2 above)
Comment (wdoug) [reply src/presentational/HelpUsImplementThis.js:14]: Good question. It stops a security vulnerability in Firefox. See [this](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/) for more info

Comment (sungbae) [reply src/presentational/HelpUsImplementThis.js:14]: This is cool! So when you click on the tooltip, does it actually direct the developer to the `featureDescriptionUrl`? Could people start to overwrite this default url to a specific issue on github per the component?

File: src/MemberProfile.js [hunk 3]
```diff
@@ -29,92 +30,66 @@ const MemberProfile = ({ user }) => {
...
+            {['Python', 'SQL', 'Enterprise Geodatabases'].map(skill => (
+              <Chip
+                key={skill}
+                className="skills-button"
+                onDelete={() => false}
+                label={skill}
+              />
+            ))}
+          </div>
+        </HelpUsImplementThis>
```
Comment (sungbae) [inline src/MemberProfile.js:74]: very nice!

File: src/MemberProfile.js [hunk 4]
```diff
@@ -29,92 +30,66 @@ const MemberProfile = ({ user }) => {
...
-          </Chip>
-        </div>
+        <HelpUsImplementThis>
+          <div className="skillsright">
+            {['Python', 'SQL', 'Enterprise Geodatabases'].map(skill => (
+              <Chip
+                key={skill}
+                className="skills-button"
+                onDelete={() => false}
+                label={skill}
```
Comment (sungbae) [inline src/MemberProfile.js:70]: Could you tell where I can see what key, onDelete(), and label is? Also shouldn't we use onRequestDelete() as above?

**Comment by sungbae** [comment]
This looks good. I just left a couple of questions.

File: src/presentational/HelpUsImplementThis.js (hunk 2 above)
Comment (wdoug) [reply src/presentational/HelpUsImplementThis.js:14]: Exactly @sungbae

File: src/MemberProfile.js (hunk 4 above)
Comment (wdoug) [reply src/MemberProfile.js:70]: The api has changed in the new version of the library: https://material-ui-next.com/api/chip/

Comment (wdoug) [reply src/MemberProfile.js:70]: material-ui-next.com is the docs site for the new version of the api

**Review by dviramontes — APPROVED** [review]
looks great! thanks @wdoug !

**Pull request merged**